### PR TITLE
Update WannaCry.md

### PR DESCRIPTION
**Pull request by brimorlabs**
Edited with changes. Trying &comma; as a tag. If that doesn't work, will have to try another html specific tag

### CODE REVIEW

File: docs/families/Hive.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+# TODO - uncomment out the following. Do not change the name.
+# parent: Ransomware Families
```
Comment (bcarrier) [inline docs/families/Hive.md]: These need to be uncommented.

File: docs/families/Hive.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+|Human Operated | No | |
+|Exfiltration | No | |
+|Propagation | No | |
+
+
+[^1]: URL https://www.microsoft.com/security/blog/2022/07/05/hive-ransomware-gets-upgrades-in-rust/
+[^2]: URL https://www.trendmicro.com/vinfo/us/security/news/ransomware-spotlight/ransomware-spotlight-hive
+[^3]: URL https://www.hhs.gov/sites/default/files/hive-ransomware-analyst-note-tlpwhite.pdf
+[^4]: URL https://www.sentinelone.com/labs/hive-attacks-analysis-of-the-human-operated-ransomware-targeting-healthcare/
+[^5]: URL https://www.microsoft.com/security/blog/2022/07/05/hive-ransomware-gets-upgrades-in-rust/
```
Comment (bcarrier) [inline docs/families/Hive.md]: We shouldn't duplicate these URLs.  5 & 6 are the same as 1.

File: docs/families/LockBit2.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+# TODO - uncomment out the following. Do not change the name.
+# parent: Ransomware Families
```
Comment (bcarrier) [inline docs/families/LockBit2.md]: Same as Hive. Need to be commented and updated.

File: docs/families/LockBit2.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+|Human Operated | No | |
+|Exfiltration | No (often uses StealBit, a seperate executable) | [^10] |
+|Propagation | Yes | [^11] |
+
+
+[^1]: URL (https://unit42.paloaltonetworks.com/lockbit-2-ransomware/)
+[^2]: URL (https://www.mandiant.com/resources/unc2165-shifts-to-evade-sanctions)
+[^3]: URL https://minerva-labs.com/blog/lockbit-2.0-ransomware-surges-in-2022/
+[^4]: URL https://www.pcrisk.com/removal-guides/21605-lockbit-2-0-ransomware
+[^5]: URL https://www.pcrisk.com/removal-guides/21605-lockbit-2-0-ransomware
```
Comment (bcarrier) [inline docs/families/LockBit2.md]: Duplicate

File: docs/families/WannaCry.md [hunk 5]
```diff
@@ -30,18 +30,14 @@ permalink: /families/wannacry/
...
-[^9]: https://www.google.com/search?q=wannacry+ransomware+yara+rules
-[^10]: https://www.csoonline.com/article/3227906/wannacry-explained-a-perfect-ransomware-storm.html
-[^11]: https://www.malwarebytes.com/wannacry
-
-
-
-
+[^1]: URL https://www.csoonline.com/article/3227906/wannacry-explained-a-perfect-ransomware-storm.html
+[^2]: URL https://www.cisa.gov/uscert/ncas/alerts/TA17-132A
+[^3]: URL https://www.csoonline.com/article/3227906/wannacry-explained-a-perfect-ransomware-storm.html
```
Comment (bcarrier) [inline docs/families/WannaCry.md]: duplicate

File: docs/families/blackcat.md [hunk 6]
```diff
@@ -1,32 +1,35 @@
...
+|Exfiltration | No | |
+|Propagation | Yes, sort of. Can spread via NetBIOS & PSExec, but only to one other system | [^8] |
 
 
+[^1]: URL https://securityaffairs.co/wordpress/132339/malware/blackcat-ransomware-clear-web.html
+[^2]: URL https://www.microsoft.com/security/blog/2022/05/09/ransomware-as-a-service-understanding-the-cybercrime-gig-economy-and-how-to-protect-yourself/#DEV-0504
+[^3]: URL https://www.microsoft.com/security/blog/2022/05/09/ransomware-as-a-service-understanding-the-cybercrime-gig-economy-and-how-to-protect-yourself/#DEV-0237
+[^4]: URL https://www.microsoft.com/security/blog/2022/06/13/the-many-lives-of-blackcat-ransomware/
+[^5]: URL https://www.varonis.com/blog/blackcat-ransomware
+[^6]: URL https://www.varonis.com/blog/blackcat-ransomware
```
Comment (bcarrier) [inline docs/families/blackcat.md]: duplicate

**Review by brimorlabs — COMMENTED** [review]
I think I got them all

**Pull request merged**